lib/assisted_workflow/addons/github.rb

```diff
@@ -22,12 +22,8 @@ def description
       @issue.body.to_s.gsub("\r\n", "\n")
     end
     
-    def other_id
```

Comment:
perhaps we can use a new configuration variable and use that like `branch_username`